css-two-column-layout/index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Two Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+    <div class="row">
+      <div class="col-full text-align-c">
+        <h1>SCALE, SPEED, AGILITY</h1>
+        <p>We've dedicated thousands of man hours to making healthcare institutions in the Helium network run 10x more efficiently and deliver an unparalleled quality of care.</p>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-one-fourth">
+        <img src="./images/software.svg" alt="Software">
+      </div>
+      <div class="col-half">
+        <h3>INTUITIVE, SMART SOFTWARE</h3>
+        <p>We build products that transform how healthcare facilities run across Africa - making them radically more efficient operationally and financially, a complete turnaround.</p>
+      </div>
```

Comment:
I would consider grouping these elements under one `column-half` and then subdividing that column to get the two parts as you have done here, since the content of these rows each take up 50% of those rows.